documentation/production/tutorial-production-mode-basic.asciidoc

```diff
@@ -84,3 +84,102 @@ When minifying code, it is often obscured, which makes it harder to read, hence
 Bundling is an optimization where we merge multiple files to a single collection so that the browser doesn't need to
 request so many files from the server. 
 This makes the application load faster.
+
+== Checklist for the production mode app artifact
+
+Vaadin Servlet configures itself from the `flow-build-info.json` file, which is located in the `META-INF/VAADIN/config/` resource package.
+The resources location differ for different artifact types, for example:
+
+. Spring Boot jar file places resources into the `BOOT-INF/classes/` folder in the jar file;
+. WAR archive places resources into the `WEB-INF/classes/` folder.
+
+If the production mode has been activated properly, the contents of the `flow-build-info.json` should be as follows:
+
+.flow-build-info.json
+[source, json]
+----
+{
+  "compatibilityMode": false,
+  "productionMode": true,
+  "enableDevServer": false,
+  "chunks": {
+    "fallback": {
+      "jsModules": [
+        "@vaadin/vaadin-icons/vaadin-icons.js",
+        "@vaadin/vaadin-grid/src/vaadin-grid-tree-toggle.js",
+// etc etc
+        "frontend://ironListConnector.js"
+      ],
+      "cssImports": [
+      ]
+    }
+  }
+}
+----
+
+(Note that the `productionMode` property is set to `true`).
+
+It's very important to have this file on classpath. If it's not, Vaadin Servlet
+will then try to auto-configure itself and sometimes wrongly so. For example when launching
+WAR in Tomcat from Intellij, the `frontendFolder` setting will be "auto-detected"
+to Tomcat's `bin/` folder, which will cause webpack to fail later on, because of missing files.
+
+If the file is present multiple times on the classpath,
+then Vaadin tries to choose which one is the right one (which one is coming
+from the main project). That's unfortunate since sometimes the algorithm fails and
```

Comment:
How is it unfortunate or backfires? If we can't 100% pick the one we want then there is a log warning/debug
`Unable to fully determine correct flow-build-info.` with the warning informing to check the content
`Accepting file '%s' first match of '%s' possible. Please verify flow-build-info file content.`

Reply:
The choice of a warning message is something I perceive as "unfortunate"; "backfires" refers to "random crashes in production". However, let us have this discussion elsewhere: https://github.com/vaadin/flow/issues/8936

For now, I agree that it's better to remove 'unfortunate' and 'backfires' and replace it with the warning message.